CONTRIBUTING.md

```diff
@@ -33,3 +33,39 @@ accept your pull requests.
    recommended coding standards for this organization.
 1. Ensure that your code has an appropriate set of unit tests which all pass.
 1. Submit a pull request.
+
+
+## Best Practices
+
+The following best-practice guidelines will help ensure your initialization
+actions are less likely to break from one Dataproc version to another, and
+most likely to support different single-node, high-availability, and
+standard cluster modes.
+
+1. Where possible, install from Dataproc's prebuilt Debian packages (built
+   using Apache Bigtop) instead of installing from tarballs. Dataproc packages
+   can be found under /var/lib/apt/lists/*dataproc-bigtop-repo_*Packages on
```

Comment:
the \*dataproc-bigtop-repo_\* is not rendering in italics. Also, do you want a space before Packages?

Reply:
"Packages" is part of the glob. Added to whole thing to backticks now.